zcash_primitives/src/transaction/mod.rs

```diff
@@ -1021,18 +1250,24 @@ pub mod testing {
             expiry_height in any::<u32>(),
             transparent_bundle in transparent::arb_bundle(),
             sapling_bundle in sapling::arb_bundle_for_version(version),
-            orchard_bundle in orchard::arb_bundle_for_version(version),
+            orchard_bundle in orchard_testing::arb_bundle_for_version(version),
+            _orchard_zsa_bundle in orchard_testing::arb_zsa_bundle_for_version(version),
+            _issue_bundle in issuance::testing::arb_bundle_for_version(version),
```

Comment:
why the `_`?

Reply:
in these proptests I'm not sure its possible to hide it behind feature flag, so I want avoid clippy bitching at unused arguments for non-nu6 builds